Force geoclue to use the "fallback" mac address fetching code.
The NM-based code does not seem to work.

Patch by Kevin McCarthy <signals@gentoo.org>

--- src/connectivity.c
+++ src/connectivity.c
@@ -218,7 +218,7 @@
 geoclue_connectivity_get_router_mac (GeoclueConnectivity *self)
 {
 	if (self == NULL ||
-	    GEOCLUE_CONNECTIVITY_GET_INTERFACE (self)->get_router_mac == NULL) {
+	    GEOCLUE_CONNECTIVITY_GET_INTERFACE (self)->get_router_mac == NULL || 1) {
 		char *mac = NULL;
 		guint i;
 		int ret_val;
